### PR TITLE
2339 - Add assessment short label

### LINKED ISSUE #2339: add assessment label
add 

- [x] in email 

> You have been invited to access **_fra_** 2025 as National correspondent for Atlantis 06.

1. fra -> Should be FRA 
2. panEuropean-> panEuropean

- [x] in country selector

1. FRA -> FRA 
2. PANEUROPEAN -> should be panEuropean

### PR DESCRIPTION
**Pull request by rfontanarosa**
close #2339

Screenshot 2023-05-08 alle 15 14 32

Screenshot 2023-05-08 alle 15 13 43

Screenshot 2023-05-08 alle 15 12 34

Screenshot 2023-05-08 alle 15 12 19

### CODE REVIEW

**Review by sorja — DISMISSED** [review]
Can it be possible we are using somewhere else in the platform 'FRA' <==> assessmentName?

File: src/client/components/AreaSelector/CountryList/CountryListRoleSection/CountryListRoleSection.tsx [hunk 1]
```diff
@@ -42,7 +42,9 @@ const CountryListRoleSection: React.FC<Props> = (props: Props) => {
       {role !== UserRoles.noRole.role && (
         <div className="country-selection-list__header">
           <span className="country-selection-list__primary-col">{i18n.t<string>(Users.getI18nRoleLabelKey(role))}</span>
-          <span className="country-selection-list__secondary-col uppercase">{`${assessment.props.name} ${cycle.name}`}</span>
+          <span className="country-selection-list__secondary-col">
+            {`${i18n.t(`${assessment.props.name}.labels.short`)} ${cycle.name}`}
```
Comment (minotogna) [inline src/client/components/AreaSelector/CountryList/CountryListRoleSection/CountryListRoleSection.tsx]: Maybe better to add a util to Assessments, like ```Assessments.getShortLabel({assessmentName})=>string``` ?

Comment (minotogna) [reply src/client/components/AreaSelector/CountryList/CountryListRoleSection/CountryListRoleSection.tsx]: I see no change.

Comment (rfontanarosa) [reply src/client/components/AreaSelector/CountryList/CountryListRoleSection/CountryListRoleSection.tsx]: sorry, there was a conflict

**Review by minotogna — APPROVED** [review]

**Review by sorja — APPROVED** [review]

**Pull request merged**